Formula/ntopng.rb

```diff
@@ -1,9 +1,17 @@
 class Ntopng < Formula
   desc "Next generation version of the original ntop"
   homepage "http://www.ntop.org/products/ntop/"
-  url "https://downloads.sourceforge.net/project/ntop/ntopng/ntopng-2.2.tar.gz"
-  sha256 "4fccfc9e9f333addcd3c957b4520c471117bc2df5655d6eabf328c7385fb255e"
-  revision 1
+
+  stable do
+    url "https://github.com/ntop/ntopng/archive/2.4.tar.gz"
+    sha256 "86f8ed46983f46bcd931304d3d992fc1af572b11e461ab9fb4f0f472429bd5dd"
+
+    resource "nDPI" do
+      # tip of 1.8-stable branch; four commits beyond the 1.8 tag
+      url "https://github.com/ntop/nDPI.git",
+      :revision => "6fb81f146e2542cfbf7fab7d53678339c7747b35"
```

Comment:
The inconsistency with those has been driving me crazy. I think the third one may have become "popular" because gdm screws them up, and then that became the supposed standard, when in fact the second one is correct since it doesn't pretend the comma isn't there, and is also superior to the first because it makes a best effort to avoid going over the 80 character column limit, whereas the first doesn't unless the URL happens to be particularly short, in which case deciding between the first and second might require a neutral third party arbitrator.